packages/serum/src/markets.json

```diff
@@ -1283,7 +1283,6 @@
     "deprecated": false,
     "address": "3d4rzwpy9iGdCZvgxcu7B1YocYffVLsQXPXkBZKt2zLc"
   },
-  {
     "name": "POLIS/USDC",
```

Comment:
not sure why this is showing up as deleted but if you check the source its there :confused: